web/src/read_receipts.ts

```diff
@@ -19,88 +19,91 @@ const read_receipts_api_response_schema = z.object({
     user_ids: z.array(z.number()),
 });
 
+export function fetch_read_receipts(message_id: number): void {
+    const message = message_store.get(message_id);
+    assert(message !== undefined, "message is undefined");
+
+    if (message.sender_email === "notification-bot@zulip.com") {
+        $("#read_receipts_modal .read_receipts_info").text(
+            $t({
+                defaultMessage: "Read receipts are not available for Notification Bot messages.",
+            }),
+        );
+        $("#read_receipts_modal .modal__content").addClass("compact");
+    } else {
+        loading.make_indicator($("#read_receipts_modal .loading_indicator"));
+        void channel.get({
+            url: `/json/messages/${message_id}/read_receipts`,
+            success(raw_data) {
+                const $modal = $("#read_receipts_modal").filter(
+                    "[data-message-id=" + message_id + "]",
+                );
+                // If the read receipts modal for the selected message ID is closed
+                // by the time we receive the response, return immediately.
+                if (!$modal.length) {
```

Comment:
This change looks like it's reverting a recent linter fix. Please undo, and be more careful when resolving merge conflicts to not drop others' changes. (`git log -p` on this file will show you the third-party changes)

Reply:
Hey @timabbott , I changed in my previous push because it was giving an eslint error and was suggesting to change it.

Reply:
Done!